src/iexamine.cpp

```diff
@@ -4916,6 +4918,23 @@ void iexamine::autodoc( player &p, const tripoint &examp )
         autodoc_header = warning +
                          _( "\n Using the Autodoc without an operator can lead to <color_light_cyan>serious injuries</color> or <color_light_cyan>death</color>.\n By continuing with the operation you accept the risks and acknowledge that you will not take any legal actions against this facility in case of an accident. " );
     }
+
+    std::vector<item> arm_splints;
+    std::vector<item> leg_splints;
+
+    for( const item &supplies : get_map().i_at( examp ) ) {
+        if( supplies.typeId() == itype_arm_splint ) {
+            arm_splints.push_back( supplies );
+        }
+        if( supplies.typeId() == itype_leg_splint ) {
+            leg_splints.push_back( supplies );
+        }
```

Comment:
Could I suggest a flag instead of a hardcoded id check?

Reply:
I thought about it, but as you stated below, it made using `use_amount` impossible, so I decided that hardcoded ids would be tolerable for now.